doc/lsst.ap.pipe/apdb.rst

```diff
@@ -64,7 +71,15 @@ Databases can also be set up using :ref:`config files <command-line-config-howto
 
    make_apdb.py -C myApdbConfig.py
    pipetask run -p ApPipe.yaml -C myApPipeConfig.py  -b repo -o myrun
+   
+A Postgres database can be set up using :ref:`bps yaml files <creating-a-yaml-file>` by adding this to their yaml:
```

Comment:
Could you clarify what you mean by `by adding this to their yaml file`? It's not clear what `their` is in this context if you just glance at this sentence.